src/Altinn.Notifications.Core/Integrations/IProfileClient.cs

```diff
@@ -13,4 +13,12 @@ public interface IProfileClient
     /// <param name="nationalIdentityNumbers">A list of national identity numbers to look up contact points for</param>
     /// <returns>A list of contact points for the provided national identity numbers </returns>
     public Task<List<UserContactPoints>> GetUserContactPoints(List<string> nationalIdentityNumbers);
+
+    /// <summary>
+    /// Retrieves the user registered contact points for a list of organization corresponding to a list of organization numbers
+    /// </summary>
+    /// <param name="organizationNumbers">The set or organizations to retrieve contact points for</param>
+    /// <param name="resourceId">The id of the resource to look up contact points for</param>
+    /// <returns>A list of organiation contact points containing user registered contact points</returns>
+    public Task<List<OrganizationContactPoints>> GetUserRegisteredOrganizationContactPoints(List<string> organizationNumbers, string resourceId);
```

Comment:
A bit of a mouthful, and other suggestions as to what we can name it ?

Reply:
1. GetUserRegisteredContactPoints (just remove organization, it's implied based on input?)
2. GetRightHoldersContactPoints (right holders can also imply a user in association with another party)
3. GetYourContactInformationForOrganization (reuse the term used by the GUI where users edit the information)

Reply:
Went for option 1 ✅